ui/mirage/factories/mfa-okta-method.js

```diff
@@ -9,7 +9,7 @@ export default Factory.extend({
   base_url: 'okta.com',
   mount_accessor: '',
   name: '', // returned but cannot be set at this time
-  namespace_id: 'root',
+  namespace_path: 'root',
```

Comment:
```suggestion
  namespace_path: 'admin/',
```


Expanding on my comment above, root technically doesn't return a `namespace_path` and actually does return on `namespace_id` (I think this is why there was confusion during the original implementation, see sample response above) can we update this to be `admin/` (or whatever namespace you'd like 😄 )